
<launch>
    <arg name="hand_name" default="pr2_left" />
    <arg name="executer_planning_time" default="20"/>
    <arg name="analyzer_planning_time" default="5"/>

    <rosparam command="load" file="$(find moveit_trajectory_planner)/configs/$(arg hand_name).grasping.yaml"/>
    <rosparam command="load" file="$(find moveit_trajectory_planner)/configs/$(arg hand_name).workspace.yaml"/>

    <node name="grasp_executer" pkg="moveit_trajectory_planner" type="grasp_execution_node.py" output="screen">
        <param name="allowed_planning_time" value="$(arg executer_planning_time)" />
    </node>
    <node name="grasp_analyzer" pkg="moveit_trajectory_planner" type="grasp_analyzer_node.py" >
        <param name="allowed_planning_time" value="$(arg analyzer_planning_time)" />
    </node>
    <node name="world_manager" pkg="moveit_trajectory_planner" type="world_manager_node.py" />

</launch>
